frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -239,6 +240,10 @@ export const dashboardLogic = kea<dashboardLogicType>({
                 return allItemsLoading || Object.values(refreshStatus).some((s) => s.loading)
             },
         ],
+        isRefreshing: [
+            () => [selectors.refreshStatus],
+            (refreshStatus) => (id: number) => !!refreshStatus[id]?.loading,
```

Comment:
This could even be documented in the kea docs as a legit pattern: selectors that return functions.